backend/tests/test_ingest_statements.py

```diff
@@ -0,0 +1,90 @@
+from composer.services.cs_ingestion.cs_ingestion_services import ingest_statements
+from composer.models import ConnectivityStatement
+import unittest
+from composer.services.cs_ingestion.neurondm_script import log_error
+from composer.enums import CSState
+from django.db.models import Q
+
+
+class TestIngestStatements(unittest.TestCase):
+    """
+    NOTE:
+    This test depends on the directory system of the Scicrunch neurondm - here - https://raw.githubusercontent.com/SciCrunch/NIF-Ontology/neurons/**/*.ttl
+    Check more details in neurondm_script.py
+    """
+
+    def flush_connectivity_statements(self):
+        ConnectivityStatement.objects.all().delete()
+
+    def test_full_and_label_import(self):
+        self.flush_connectivity_statements()
+
+        # Check for invalid full and label imports
+        self.assertEqual(ConnectivityStatement.objects.count(), 0)
+        ingest_statements(full_imports=['full'], label_imports=['label'])
+        self.assertEqual(ConnectivityStatement.objects.count(), 0)
+
+        # check for invalid full imports and valid label imports
+        ingest_statements(
+            full_imports=['full'],
+            label_imports=['apinatomy-neuron-populations']
+        )
+        self.assertEqual(ConnectivityStatement.objects.count(), 0)
+
+        # Check for valid full imports and invalid label imports
+        ingest_statements(full_imports=['sparc-nlp'], label_imports=['label'])
+        self.assertNotEqual(ConnectivityStatement.objects.count(), 0)
+
+        self.flush_connectivity_statements()
+
+        # check for both valid full and label imports
+        ingest_statements(
+            full_imports=['sparc-nlp'],
+            label_imports=['apinatomy-neuron-populations']
+        )
+        self.assertNotEqual(ConnectivityStatement.objects.count(), 0)
```

Comment:
suggestion (non-blocking): I don't think "at least 1 statement was ingested" is equivalent to "the ingestion is working as expected". It's a good indication but not a guarantee. My proposal is to create a new card to improve the tests by, for example, mocking the SciCrunch data files and set the exact number of statements we expect to be ingested on each test

Reply:
I agree that it doesn't mean - "the ingestion is working as expected", but this test is to test - given the right set of full and label imports, the `ingest_statements()` ingests the statements. If the labels are wrong inputs, then it doesn't import. 

Question: @afonsobspinto - do you think - the statement count would be a constant? or how often would it be changed from the Scicrunch side - in the official source?

Reply:
Had a chat with @ddelpiano! Will add a card to tech debt for further tests on checking counts.